test/kwokctl/kwokctl_restart_test.sh

```diff
@@ -81,6 +81,8 @@ function test_restart() {
     echo "Error: cluster ${name} stop error"
     return 1
   fi
+  # Wait for successful stop to prevent port occupation
+  sleep 5
```

Comment:
```suggestion
```

Currently the stop is supposed to wait for the container to exit completely, so this should not be necessary